src/Typography/Typography.stories.tsx

```diff
@@ -64,4 +106,35 @@ storiesOf('Components/Typography', module)
                 </div>
             </StyledStory>
         </ThemeProvider>
-    ));
+    ))
+    .add('Knobus Propus', () => {
+        const content = text(
+            'text',
+            'The quick brown fox jumps over the lazy dog.'
```

Comment:
The quick red fox jumps over the brown lazy dog. That sentence uses every letter in the American alphabet.

Reply:
The `red` is unnecessary. Speaking of red, did you know that strawberr......